gapic-common/lib/gapic/grpc/errors.rb

```diff
@@ -0,0 +1,63 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "googleauth"
+require "gapic/common/error"
+
+module Gapic
+  # It's called GRPCName because otherwise I get naming errors
+  # where everything referencing GRPC::Anything will try and fail to 
+  # find it here. 
+  # I've elected to use this as a stopgap and do namespace disambiguation later
+  module GRPCName
+    # Just a baseline class
+    class GRPCError < Gapic::Common::Error # or maybe GRPC::BadStatus?
```

Comment:
I think the intent is to wrap the underlying GRPC errors, right? So `Gapic::Common::Error` seems like a reasonable base class.

Reply:
:+1: